corehq/apps/receiverwrapper/tests/test_submissions.py

```diff
@@ -166,6 +171,28 @@ def list_attachments(form):
             list_attachments(new_form),
             [("file", b"text file"), ("image", b"other fake image")])
 
+    def test_submit_invalid_attachment_size(self):
+        file_data = BytesIO(b"a" * (settings.MAX_UPLOAD_SIZE_ATTACHMENT + 1))
```

Comment:
Good point! I considered this however ended up using the original settings since the file size is not that big i.e 15MB and should not impact performance much.